polyfill/index.d.ts

```diff
@@ -943,6 +943,8 @@ export namespace Temporal {
     readonly inLeapYear: boolean;
     equals(other: Temporal.PlainDateTime | DateTimeLike | string): boolean;
     with(dateTimeLike: DateTimeLike, options?: AssignmentOptions): Temporal.PlainDateTime;
+    withTime(timeLike?: Temporal.PlainTime | TimeLike | string): Temporal.PlainDateTime;
```

Comment:
Should be `withPlainTime`/`withPlainDate`?

Reply:
#1184

Reply:
Ugh, yes. I'll send a follow-up PR shortly to fix this.